meshroom/nodes/aliceVision/ImageProcessing.py

```diff
@@ -132,6 +132,15 @@ class ImageProcessing(desc.AVCommandLineNode):
                 uid=[0],
                 enabled=lambda node: node.lensCorrection.lensCorrectionEnabled.value,
             ),
+            desc.BoolParam(
+                name='chromaticAberration',
+                label='Chromatic Aberration',
+                description='Chromatic aberration (fringing) correction if model parameters are available in metadata.',
+                value=False,
+                uid=[0],
+                enabled=False  # To replace with the line below when the correction of chromatic aberration will be available
+                # enabled=lambda node: node.lensCorrection.lensCorrectionEnabled.value
```

Comment:
Enabling the correction will require also that the LCP file contains a model for defringing. That's not mandatory, only the geometric distortion model is mandatory in a LCP.